android/gradle.properties

```diff
@@ -10,7 +10,7 @@
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx10248m -XX:MaxPermSize=256m
-# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```

Comment:
This seems necessary after adding the Plaid SDK there's some more information here about this:

https://medium.com/google-developers/faster-android-studio-builds-with-dex-in-process-5988ed8aa37e

Reply:
TL;DR is that without this enabled the android app wouldn't build at all.

Reply:
JDK 8 I believe, I don't know much about Java though I just uncommented this line out based on the advice in that blog post.